libraries/chain/include/eosio/chain/authority.hpp

```diff
@@ -116,20 +116,19 @@ struct shared_key_weight {
       return key_weight{key.to_public_key(), weight};
    }
 
-   static shared_key_weight convert(chainbase::allocator<char> allocator, const key_weight& k) {
-      return std::visit(overloaded {
-         [&](const auto& k1r1) {
-            return shared_key_weight(k1r1, k.weight);
+   shared_key_weight(const key_weight& k) : key(fc::ecc::public_key_shim()), weight(k.weight) {
```

Comment:
`explicit`

Reply:
Leaving as-is because the type of the constructor parameter leave little chance for unexpected conversions.